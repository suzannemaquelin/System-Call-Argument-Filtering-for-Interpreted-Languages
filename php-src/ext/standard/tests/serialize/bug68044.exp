Warning: unserialize(): %s in %sbug68044.php on line %d

Warning: Insufficient data for unserializing - %d required, 1 present in %s%ebug68044.php on line 2

Notice: unserialize(): Error at offset 32 of 33 bytes in %s%ebug68044.php on line 2
===DONE==