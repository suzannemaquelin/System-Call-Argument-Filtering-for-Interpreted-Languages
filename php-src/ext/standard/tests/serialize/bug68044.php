<?php
	echo unserialize('C:3:"XYZ":18446744075857035259:{}');
?>
===DONE==
