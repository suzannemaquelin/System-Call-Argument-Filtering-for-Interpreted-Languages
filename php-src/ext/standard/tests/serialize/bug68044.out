Failed loading /opt/php-7.1/lib/php/extensions/debug-non-zts-20160303/xdebug.so:  /opt/php-7.1/lib/php/extensions/debug-non-zts-20160303/xdebug.so: cannot open shared object file: No such file or directory

Warning: unserialize(): Numerical result out of range in /root/php-src/ext/standard/tests/serialize/bug68044.php on line 2

Warning: Insufficient data for unserializing - 9223372036854775807 required, 1 present in /root/php-src/ext/standard/tests/serialize/bug68044.php on line 2

Notice: unserialize(): Error at offset 32 of 33 bytes in /root/php-src/ext/standard/tests/serialize/bug68044.php on line 2
===DONE==